ccl/changefeedccl: TestChangefeedMVCCTimestampWithQueries failed

ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries [failed](https://mesolite.cluster.engflow.com/invocations/default/5e9ac53e-da08-4bc6-b8d6-9f52b02f321c?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [dda4376b4d2353f7a36a505e0fe42030ecab6fc2](https://github.com/cockroachdb/cockroach/commits/dda4376b4d2353f7a36a505e0fe42030ecab6fc2):

Fatal error:

```
panic: interface conversion: interface {} is nil, not map[string]interface {} [recovered]
	panic: interface conversion: interface {} is nil, not map[string]interface {}

```
Stack: 

```
goroutine 125962 [running]:
testing.tRunner.func1.2({0x6bfe700, 0xc00ff0cf00})
	GOROOT/src/testing/testing.go:1632 +0x230
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1635 +0x35e
panic({0x6bfe700?, 0xc00ff0cf00?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries.func1(_, {0xc010dfb110, {0x7fbedde79730, 0xc0113e8620}, {{0xd336350, 0xd336370, {0x1}}, {0xd336350}, 0x0}, {{0x8719780, ...}, ...}}, ...)
	pkg/ccl/changefeedccl/changefeed_test.go:11570 +0x2e5
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamed.func1(_, {{0xc010dfb110, {0x7fbedde79730, 0xc0113e8620}, {{0xd336350, 0xd336370, {...}}, {0xd336350}, 0x0}, {{0x8719780, ...}, ...}}, ...}, ...)
	pkg/ccl/changefeedccl/helpers_test.go:1295 +0x56
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamedWithSystem.func1(0xc00d788ea0)
	pkg/ccl/changefeedccl/helpers_test.go:1330 +0x2c7
testing.tRunner(0xc00d788ea0, 0xc0106e0c30)
	GOROOT/src/testing/testing.go:1690 +0xf4
created by testing.(*T).Run in goroutine 125957
	GOROOT/src/testing/testing.go:1743 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedMVCCTimestampWithQueries
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedMVCCTimestampWithQueries3515074895
    test_log_scope.go:76: use -show-logs to present logs inline
--- FAIL: TestChangefeedMVCCTimestampWithQueries (0.68s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedMVCCTimestampWithQueries.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-50737

ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries [failed](https://tanzanite.cluster.engflow.com/invocations/default/ffaf6220-9657-4ced-9f2d-92218e76ab8a?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [90ba8536dd841563e336625eded9570fc36edc5c](https://github.com/cockroachdb/cockroach/commits/90ba8536dd841563e336625eded9570fc36edc5c):

Fatal error:

```
panic: interface conversion: interface {} is nil, not map[string]interface {} [recovered]
	panic: interface conversion: interface {} is nil, not map[string]interface {}

```
Stack: 

```
goroutine 179815 [running]:
testing.tRunner.func1.2({0x6bfe8c0, 0xc01423b1d0})
	GOROOT/src/testing/testing.go:1632 +0x230
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1635 +0x35e
panic({0x6bfe8c0?, 0xc01423b1d0?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries.func1(_, {0xc00714f2b0, {0x887c4b0, 0xc01196f090}, {{0xc002587bf0, 0xc002587c08, {0xa}}, {0xc002587bf0}, 0x0}, {{0x0, ...}, ...}}, ...)
	pkg/ccl/changefeedccl/changefeed_test.go:11570 +0x2e5
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamed.func1(_, {{0xc00714f2b0, {0x887c4b0, 0xc01196f090}, {{0xc002587bf0, 0xc002587c08, {...}}, {0xc002587bf0}, 0x0}, {{0x0, ...}, ...}}, ...}, ...)
	pkg/ccl/changefeedccl/helpers_test.go:1295 +0x56
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamedWithSystem.func1(0xc008e75380)
	pkg/ccl/changefeedccl/helpers_test.go:1330 +0x2c7
testing.tRunner(0xc008e75380, 0xc00b5fdf20)
	GOROOT/src/testing/testing.go:1690 +0xf4
created by testing.(*T).Run in goroutine 179810
	GOROOT/src/testing/testing.go:1743 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedMVCCTimestampWithQueries
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedMVCCTimestampWithQueries3674795562
    test_log_scope.go:76: use -show-logs to present logs inline
--- FAIL: TestChangefeedMVCCTimestampWithQueries (1.75s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedMVCCTimestampWithQueries.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries [failed](https://tanzanite.cluster.engflow.com/invocations/default/de42fd27-e556-4b20-981c-45b568466780?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [1a847b18290d439c62ff739ebdae00c851d5442e](https://github.com/cockroachdb/cockroach/commits/1a847b18290d439c62ff739ebdae00c851d5442e):

Fatal error:

```
panic: interface conversion: interface {} is nil, not map[string]interface {} [recovered]
	panic: interface conversion: interface {} is nil, not map[string]interface {}

```
Stack: 

```
goroutine 30097026 [running]:
testing.tRunner.func1.2({0x6c067e0, 0xc0024727e0})
	GOROOT/src/testing/testing.go:1632 +0x230
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1635 +0x35e
panic({0x6c067e0?, 0xc0024727e0?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries.func1(_, {0xc024370270, {0x887a870, 0xc0141205a0}, {{0xc01963fa40, 0xc01963fa58, {0xa}}, {0xc01963fa40}, 0x0}, {{0x0, ...}, ...}}, ...)
	pkg/ccl/changefeedccl/changefeed_test.go:11570 +0x2e5
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamed.func1(_, {{0xc024370270, {0x887a870, 0xc0141205a0}, {{0xc01963fa40, 0xc01963fa58, {...}}, {0xc01963fa40}, 0x0}, {{0x0, ...}, ...}}, ...}, ...)
	pkg/ccl/changefeedccl/helpers_test.go:1295 +0x56
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamedWithSystem.func1(0xc029221380)
	pkg/ccl/changefeedccl/helpers_test.go:1330 +0x2c7
testing.tRunner(0xc029221380, 0xc029bcd6b0)
	GOROOT/src/testing/testing.go:1690 +0xf4
created by testing.(*T).Run in goroutine 30097001
	GOROOT/src/testing/testing.go:1743 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedMVCCTimestampWithQueries
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedMVCCTimestampWithQueries939041746
    test_log_scope.go:76: use -show-logs to present logs inline
--- FAIL: TestChangefeedMVCCTimestampWithQueries (6.94s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedMVCCTimestampWithQueries.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries [failed](https://tanzanite.cluster.engflow.com/invocations/default/c76851f3-cc53-4277-b076-d41c56c069e7?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [1a847b18290d439c62ff739ebdae00c851d5442e](https://github.com/cockroachdb/cockroach/commits/1a847b18290d439c62ff739ebdae00c851d5442e):

Fatal error:

```
panic: interface conversion: interface {} is nil, not map[string]interface {} [recovered]
	panic: interface conversion: interface {} is nil, not map[string]interface {}

```
Stack: 

```
goroutine 89484 [running]:
testing.tRunner.func1.2({0xade0360, 0xc0079ba2d0})
	GOROOT/src/testing/testing.go:1632 +0x3fc
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1635 +0x6b6
panic({0xade0360?, 0xc0079ba2d0?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedMVCCTimestampWithQueries.func1(_, {0xc00739f1e0, {0x7f205953db98, 0xc0054005b0}, {{0x120e9810, 0x120e9830, {0x1}}, {0x120e9810}, 0x0}, {{0xd0c9320, ...}, ...}}, ...)
	pkg/ccl/changefeedccl/changefeed_test.go:11570 +0x466
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamed.func1(_, {{0xc00739f1e0, {0x7f205953db98, 0xc0054005b0}, {{0x120e9810, 0x120e9830, {...}}, {0x120e9810}, 0x0}, {{0xd0c9320, ...}, ...}}, ...}, ...)
	pkg/ccl/changefeedccl/helpers_test.go:1295 +0x94
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.cdcTestNamedWithSystem.func1(0xc00a11c340)
	pkg/ccl/changefeedccl/helpers_test.go:1330 +0x48e
testing.tRunner(0xc00a11c340, 0xc001b443c0)
	GOROOT/src/testing/testing.go:1690 +0x227
created by testing.(*T).Run in goroutine 89479
	GOROOT/src/testing/testing.go:1743 +0x826
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedMVCCTimestampWithQueries
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedMVCCTimestampWithQueries3559429463
    test_log_scope.go:76: use -show-logs to present logs inline
--- FAIL: TestChangefeedMVCCTimestampWithQueries (5.54s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedMVCCTimestampWithQueries.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -11558,7 +11558,7 @@ func TestChangefeedMVCCTimestampWithQueries(t *testing.T) {
 		sqlDB.Exec(t, `CREATE TABLE foo (key INT PRIMARY KEY);`)
 		sqlDB.Exec(t, `INSERT INTO foo VALUES (1);`)
 
-		feed, err := f.Feed(`CREATE CHANGEFEED WITH mvcc_timestamp AS SELECT * FROM foo`)
+		feed, err := f.Feed(`CREATE CHANGEFEED WITH mvcc_timestamp, format=json, envelope=bare AS SELECT * FROM foo`)
 		require.NoError(t, err)
 		defer closeFeed(t, feed)
 

